static/src/javascripts/projects/journalism/modules/get-campaign.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import config from 'lib/config';
+
+export const getCampaign = () => {
```

Comment:
Let's add Flow annotations everywhere ⚡️ I can help you with that. Also, it is probably a good idea to add some tests.